src/app/components/elements/UserList.jsx

```diff
@@ -23,18 +23,19 @@ class UserList extends React.Component {
 
     render() {
         const { state: { historyIndex } } = this;
+        const account = this.props.account;
         const users = this.props.users;
         const title = this.props.title;
 
         let idx = 0;
         let user_list = users.map(user => (
-            <UserListRow user={user} key={idx++} />
+            <UserListRow account={account} user={user} key={idx++} />
         ));
+        user_list = user_list.toArray();
```

Comment:
rather than `let`ing `user_list` we should make it a `const` and chain the `toArray` call after the `map`:

```
const user_list = users.map(blahblah).toArray()
```